MFVNeutralino/python/Vertexer_cfi.py

```diff
@@ -26,9 +28,11 @@
                                no_track_cuts = cms.bool(False),
                                min_seed_jet_pt = cms.double(30),
                                min_track_pt = cms.double(1),
+                               min_track_pt_loose = cms.double(0.9),
                                min_track_dxy = cms.double(0),
                                min_track_sigmadxy = cms.double(0),
                                min_track_rescaled_sigmadxy = cms.double(4),
+                               min_track_rescaled_sigmadxy_loose = cms.double(3.5),   
```

Comment:
Is this used anywhere, or can we drop it?

Reply:
No as I far as I know. So, yes I will drop it.

Reply:
I'm removing it now--thanks for confirming!